common/common-api/src/main/java/org/ow2/proactive/scripting/Script.java

```diff
@@ -386,26 +390,39 @@ public void close() throws IOException {
             prepareBindings(bindings);
             Object evalResult = engine.eval(getReader());
 
-            engine.getContext().getErrorWriter().flush();
-            engine.getContext().getWriter().flush();
-
             // Add output to the script result
             ScriptResult<E> result = this.getResult(evalResult, bindings);
-            result.setOutput(outputBoundedWriter.toString());
+            captureOutput(engine, outputBuffer, result);
 
             return result;
         } catch (javax.script.ScriptException e) {
+            logger.error("Error during script execution", e);
```

Comment:
I'm afraid this logger will print error messages everywhere. Not sure it's useful as we return a ScriptResult

Reply:
Ok I removed it.